torchmultimodal/models/video_vqvae.py

```diff
@@ -14,47 +14,60 @@
 from torchmultimodal.utils.assertion import assert_equal_lengths
 from torchmultimodal.utils.common import to_tuple_tuple
 
-DEFAULT_ENCODER_IN_CHANNEL_DIMS = (3, 240, 240, 240, 240, 240)
-DEFAULT_DECODER_OUT_CHANNEL_DIMS = (240, 240, 240, 240, 3)
-
 
 def video_vqvae(
```

Comment:
Consider renaming as "[video_vqvae_res64_ds222](https://github.com/wilson1yan/VideoGPT-Paper/blob/master/videogpt/config_model.py#L5)" since this is just one of the variants from the original paper?

Reply:
at that point do we even need this builder?